queries.sql

```diff
@@ -7,4 +7,71 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/* Second entry*/
+
+-- Transaction 1
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+TABLE animals;
```

Comment:
- [ OPTIONAL ] If you want to check here you could check by this and select the species so you can try this command!🔥💯
> SELECT species from animals;